client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -101,7 +101,7 @@
 	color: var( --contrastColor );
 	font-size: 14px;
 	// stylelint-disable-next-line scales/font-weight
-	font-weight: 500;
+	font-weight: 600;
```

Comment:
Note the Stylelint disable above this line should also be removed with the change.

@dubielzyk can you review the visual change in the "I don't know" button, please? Should we go with font-weight 400 or 600?

<img width="1007" alt="Screenshot 2020-05-13 at 09 28 02" src="https://user-images.githubusercontent.com/87168/81778763-1a2c4600-94fc-11ea-80bb-6e91597d9849.png">
<img width="982" alt="Screenshot 2020-05-13 at 09 28 09" src="https://user-images.githubusercontent.com/87168/81778759-18fb1900-94fc-11ea-983e-c2191df66edb.png">
<img width="993" alt="Screenshot 2020-05-13 at 09 29 21" src="https://user-images.githubusercontent.com/87168/81778851-3cbe5f00-94fc-11ea-97f6-aa76b6ddaaab.png">

Reply:
I really prefer 500, but I'm fine with going for 600 if we wanna stick with the system

Reply:
Hey hey! I haven't finished going through this yet, I just did a find and replace through `/client`. My intent is to leave alone any weights that don't apply to sans-serif fonts so I'm fine switching this back to 500. :)

Reply:
This would be sans-serif; it's the button in the screenshot, not the serif big text. 👍 

Sorry if I jumped in this early, I just got a ping via code-owner tagging. :-)

Reply:
Ohhh gotcha. No worries, thanks for checking in! Yeah, we'd like to keep to 600 across the board if possible.